fedimint-core/src/envs.rs

```diff
@@ -18,6 +18,9 @@ pub const FM_USE_UNKNOWN_MODULE_ENV: &str = "FM_USE_UNKNOWN_MODULE";
 
 pub const FM_ENABLE_MODULE_LNV2_ENV: &str = "FM_ENABLE_MODULE_LNV2";
 
+/// Print sensitive secrets without redacting them. Use only for debugging.
+pub const FM_DEBUG_SHOW_SECRETS_ENV: &str = "FM_DEBUG_SHOW_SECRETS";
```

Comment:
Probably ok to add as ad-hoc env var since it's purely for debugging, so us devs is the only target audience.